### PR TITLE
fixed image preview

### PR DESCRIPTION
**Pull request by myroslav-melnychenko**

### CODE REVIEW

File: src/app/shared/components/image-form-control/image-form-control.component.html [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
-      </div>
-      <ng-container *ngFor="let img of decodedImages;">
-        <div class="logo_images">
-          <img src="{{img}}" alt="">
-          <mat-icon class="logo_cancel" (click)="onRemoveImg(img)">cancel</mat-icon>
-        </div>
-      </ng-container>
-    </div>
+    <mat-grid-list cols="4" rowHeight="58px" class="logo" [gutterSize]="'8px'">
+        <mat-grid-tile  class="logo_button" *ngIf="selectedImages.length < imgMaxAmount || selectedImages.length === 0">
```
Comment (dmitryy90) [inline src/app/shared/components/image-form-control/image-form-control.component.html:6]: indentations

File: src/app/shared/components/image-form-control/image-form-control.component.scss [hunk 2]
```diff
@@ -15,26 +15,27 @@
   font-weight: normal;
   font-size: 0.813rem;
   line-height: 1.125rem;
-  margin:0.3rem;
+  margin-bottom:0.3rem;
```
Comment (dmitryy90) [inline src/app/shared/components/image-form-control/image-form-control.component.scss:18]: indentations

File: src/app/shared/components/image-form-control/image-form-control.component.scss [hunk 3]
```diff
@@ -54,17 +55,27 @@
...
-    background: #FFFFFF;
+     width: 95px;
+     height: 58px;
+     background: #FFFFFF;
       img {
-        border-radius: 0.313rem;
         object-fit: cover;
-        width: 100%;
-        height: 100%;
+        width: 95px;
```
Comment (dmitryy90) [inline src/app/shared/components/image-form-control/image-form-control.component.scss:63]: use relative units

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Comment by litvinets** [comment]
Unfortunately, this task is not completed. The result does not meet the expectations. It should look like this: 
image

But currently, you did this: 
image

In addition, the delete icons should be acknowledged by the BA and designers. At least, I think it has to be improved (remove these gray borders, and probably the red color is too aggressive). It should be discussed. 
image


In the row should be only 4 pictures that evenly stretched in the layout.  
image

Something happened to the limitation of pictures, for example in this case it should be only 10 pictures max, but not 13. Verify this bug, please.

**Comment by litvinets** [comment]
I see that input for new pictures disappeared when you reach the limit, but if you already clicked on it you can still attach as many pictures as you want. So, this is a problem.

File: src/app/shared/components/image-form-control/image-form-control.component.scss [hunk 4]
```diff
@@ -54,17 +55,27 @@
...
+.logo_cancel:hover {
+  display: block
+}
+mat-icon {
+  background: rgba(white, 0.3);
+  color:black;
+}
+mat-icon:hover{
+  color:red;
+}
```
Comment (litvinets) [inline src/app/shared/components/image-form-control/image-form-control.component.scss:81]: you can try using nesting
mat-icon{
  &:hover{ }
}

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shared/components/image-form-control/image-form-control.component.ts [hunk 5]
```diff
@@ -29,6 +29,13 @@ export class ImageFormControlComponent implements OnInit {
   constructor() { }
 
   ngOnInit(): void {
+    if(window.innerWidth >= 500){
```
Comment (dmitryy90) [inline src/app/shared/components/image-form-control/image-form-control.component.ts:32]: use media queries

File: src/app/shared/components/image-form-control/image-form-control.component.ts [hunk 6]
```diff
@@ -92,5 +103,14 @@ export class ImageFormControlComponent implements OnInit {
   setDisabledState(disabled: boolean) {
     this.disabled = disabled;
   }
-
+  /* This method controls cols quantity in the img preview grid rows depending on screen width */
+  onResize(event) {
+    if(event.target.innerWidth >= 500){
```
Comment (dmitryy90) [inline src/app/shared/components/image-form-control/image-form-control.component.ts:108]: duplicated code

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**